src/Component.php

```diff
@@ -240,10 +240,21 @@ private function handleFor( DOMNode $node, array $data ) {
 			list( $itemName, $listName ) = explode( ' in ', $node->getAttribute( 'v-for' ) );
 			$node->removeAttribute( 'v-for' );
 
-			foreach ( $data[$listName] as $item ) {
+			if ( preg_match( '/\((?P<itemName>[^,]+)\s*,\s*(?P<keyName>[^\)]+)\)/', $itemName, $matches ) ) {
```

Comment:
Note that this uses an *extremely* crude regex to match variable names; I’m not sure if this library has a better regex (snippet) for that elsewhere, though.